app.py

```diff
@@ -472,6 +503,36 @@ def invoice():
     #TODO: other statuses..
     return render_template("invoice.html", invoice=invoice, order=order, error=error, qrcode_svg=qrcode_svg, url=url)
 
+@app.route('/bronze_oauth')
+def bronze_oauth():
+    if not bronze.authorized:
```

Comment:
move oauth resources (url and userinfo etc) into the global context, something like:

```
@app.before_request
def before_request_func():
    g.bronze_kyc_url = XXXX
    g.bronze_userinfo = XXXX
```

This way we can set it once and it can be used in every template via the parent layout.html template

Reply:
not sure how to do that.

Reply:
added some stuff to this.